docs/aop/user/apps/usercentrics.md

```diff
@@ -0,0 +1,138 @@
+```
+
+where `apXtrZ9ec` is the setting ID that identifies configuration of the data processing services for your store in the Usercentrics Admin Interface.
+
+For the direct integration, you have to programmatically deactivate the JavaScript code of all the tracking tools on the shop pages and give tools the names that match those in the Usercentrics Admin Interface.
+   
+For more details on the Usercetrics direct integration, see the [Usercentrics documentation](https://docs.usercentrics.com/#/direct-implementation-guide).
+
+#### Smart Data Protector
+
+The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) blocks the data processing services you added to your store and activates them only for customers that gave their consent to do so. The Smart Data Protector is the preferred and default setting to configure Usercentrics using Spryker. You can apply it if you don't use a third-party tracking tool yet and don't want to use developers' work.
+
+For this setting, Usercentrics JavaScript tag has to be injected with the setting ID and Smart Data Protector JavaScript code. Once you set up data processing services and got the setting ID for your app from the Usercentrics Admin Interface, the Smart Data Protector automatically manages all the tracking tools, and no manual work is required from your side.
+
+{% info_block infoBox "Info" %}
+
+Every data processing service not supported by Usercentrics by default requires a custom manual configuration in the Usercentrics Admin Interface and corresponding adaption as for the [Direct Integration](#direct-integration) case. For details on the custom services, see [Usercentrics documentation on custom data processing services](https://usercentrics.atlassian.net/servicedesk/customer/portal/2/article/185794627).
+
+{% endinfo_block %}
+
+#### Google Tag Manager
+
+If you already have the [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) integrated inещ your store, to use the technology legally, you need the cookie consent popup to appear on your Storefront. You can achieve that by using the Google Tag Manager with Usercentrics.
+
+To use Usercentrics with the Google Tag Manager, make sure that:
+
+- The Usersentrics data processing services match the tracking tools you have in the Google Tag Manager. For this, in the Usercentrics Admin Interface, go to **Service Settings -> Data Processing Services** and define the same data processing services that you have in the Google Tag Manager.
+
+- In the Google Tag Manager UI, configure the variables and triggers to work with Usercentrics. For details on how to do that, see [Google Tag Manager Configuration](https://docs.usercentrics.com/#/browser-sdk-google-tag-manager-configuration).
+
+{% info_block infoBox "Info" %}
+
+You can use the Google Tag Manager with Usercentrics Smart Data Protector or via direct integration of Usercentrics.
```

Comment:
@thomas-schwaier-spryker  I'm not sure if this statement is correct
Was it taken from Usercentrics documentation?

@lenadoc , could you please advise?

Reply:
I have taken it from here: https://spryker.aha.io/features/AS-100 (_Google Tag Manager can be used with UserCentrics Smart Data Protector or the simple direct integration of User Centrics_) @thomas-schwaier-spryker should we keep this sentence or better delete it?

Reply:
@lenadoc please remove it as I trust in @lenadoc judgment here, as he has implemented it and I'm not exactly sure why I wrote it, as there is also no mention in the UserCentrics documentation for Google Tag Manager.